apps/frontend/src/components/template/QuestionTemplateLabel.tsx

```diff
@@ -0,0 +1,22 @@
+import makeStyles from '@mui/styles/makeStyles';
+import React from 'react';
+
+export default function TemplateEditLabel(props: { pageType: string }) {
+  const useStyles = makeStyles((theme) => ({
+    label: {
+      color: theme.palette.primary.main,
+      backgroundColor: theme.palette.grey[300],
+      fontSize: 'medium',
+    },
+  }));
+  const classes = useStyles();
+  if (props.pageType === 'Template') {
+    return (
+      <label className={classes.label}>
+        You are editing the question as it appears on the current template
+      </label>
+    );
+  }
+
+  return <label className={classes.label}>You are editing the question</label>;
```

Comment:
Instead of using `makeStyles` we can try to use the `FormLabel` MUI component with `sx` property where we can pass the custom styles that we need. https://mui.com/material-ui/api/form-label/